image_geometry/CMakeLists.txt

```diff
@@ -1,38 +1,54 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.5)
 project(image_geometry)
 
-find_package(catkin REQUIRED sensor_msgs)
+find_package(ament_cmake REQUIRED)
+
+if(NOT WIN32)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14 -Wall -Wextra")
+endif()
+
 find_package(OpenCV REQUIRED)
+find_package(sensor_msgs REQUIRED)
+find_package(Boost REQUIRED)
 
-catkin_package(CATKIN_DEPENDS sensor_msgs
-               DEPENDS OpenCV
-               INCLUDE_DIRS include
-               LIBRARIES ${PROJECT_NAME}
-)
+# catkin_package(CATKIN_DEPENDS sensor_msgs
+#                DEPENDS OpenCV
+#                INCLUDE_DIRS include
+#                LIBRARIES ${PROJECT_NAME}
+# )
+
+# catkin_python_setup()
 
-catkin_python_setup()
+#include_directories(${catkin_INCLUDE_DIRS}
+#                           ${OpenCV_INCLUDE_DIRS}
+#)
+#include_directories(include)
 
-include_directories(${catkin_INCLUDE_DIRS}
-                           ${OpenCV_INCLUDE_DIRS}
+include_directories(include
+                    ${OpenCV_INCLUDE_DIRS}
 )
-include_directories(include)
 
 # add a library
 add_library(${PROJECT_NAME} src/pinhole_camera_model.cpp src/stereo_camera_model.cpp)
 target_link_libraries(${PROJECT_NAME} ${OpenCV_LIBRARIES})
-add_dependencies(${PROJECT_NAME} ${catkin_EXPORTED_TARGETS})
+ament_target_dependencies(${PROJECT_NAME} "sensor_msgs" "OpenCV")
```

Comment:
Nit: please use separate line when you add a list of dependencies.
also we usually list dependencies alphabetically